src/Symfony/Component/Validator/Resources/translations/validators.ro.xlf

```diff
@@ -334,6 +334,58 @@
                 <source>This value should be valid JSON.</source>
                 <target>Această valoare trebuie să fie un JSON valid.</target>
             </trans-unit>
+            <trans-unit id="87">
+                <source>This collection should contain only unique elements.</source>
+                <target>Acest set ar trebui să conțină numai elemente unice</target>
+            </trans-unit>
+            <trans-unit id="88">
+                <source>This value should be positive.</source>
+                <target>Această valoare ar trebui să fie pozitivă.</target>
+            </trans-unit>
+            <trans-unit id="89">
+                <source>This value should be either positive or zero.</source>
+                <target>Această valoare trebuie să fie pozitivă sau zero.</target>
+            </trans-unit>
+            <trans-unit id="90">
+                <source>This value should be negative.</source>
+                <target>Această valoare ar trebui să fie negativă.</target>
+            </trans-unit>
+            <trans-unit id="91">
+                <source>This value should be either negative or zero.</source>
+                <target>Această valoare trebuie să fie negativă sau zero.</target>
+            </trans-unit>
+            <trans-unit id="92">
+                <source>This value is not a valid timezone.</source>
+                <target>Această valoare nu este un fus orar valid.</target>
+            </trans-unit>
+            <trans-unit id="93">
+                <source>This password has been leaked in a data breach, it must not be used. Please use another password.</source>
+                <target>Această parolă a fost compromisă și nu poate fi utilizată. Vă rugăm să utilizați o altă parolă.</target>
+            </trans-unit>
+            <trans-unit id="94">
+                <source>This value should be between {{ min }} and {{ max }}.</source>
+                <target>Această valoare trebuie să fie între {{ min }} și {{ max }}.</target>
+            </trans-unit>
+            <trans-unit id="95">
+                <source>This value is not a valid hostname.</source>
+                <target>Această valoare nu este un domeniu valid.</target>
```

Comment:
I know I translated this to Swedish before. I also translated "hostname" to "domain" because it sounded better. But if I rememeber correctly, @javiereguiluz corrected me and suggested that I should be technical and use "hostname". 

Does it makes more sense to use "numele gazdei" here?

Reply:
Thanks for the review. Unfortunately, I can't answer that question as I do not speak romanian. The translations come from an external language translator, anyway, I have changed it to `numele gazdei` as that's what can be found online for `hostname`. 😄

Reply:
“external language translator” as in “human” or “computer”?

Reply:
Should have been more clear about that. Human, of course.